docs/01-app/03-api-reference/02-components/image.mdx

```diff
@@ -412,8 +412,9 @@ You can also [generate a solid color Data URL](https://png-pixel.com) to match t
 unoptimized = {false} // {false} | {true}
 ```
 
-When true, the source image will be served as-is instead of changing quality,
-size, or format. Defaults to `false`.
+When true, the source image will be served as-is from the `src` instead of changing quality, size, or format. Defaults to `false`.
+
+This is useful for images that do not benefit from optimization such as small images (<1KB), vector images (SVG), or animated images (GIF).
```

Comment:
The `<` caused a parser error. Fixing in https://app.graphite.dev/github/pr/vercel/next.js/73658/docs-Fix-image-component-API-reference-parsing

Reply:
Nice catch, thanks!

That link didn't work for me but I found the PR here:

- https://github.com/vercel/next.js/pull/73658